cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +184,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content []byte
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
+	errors = []error{}
+	f, err := os.Open(fcc.Path)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%s does not exists", fcc.Path))
```

Comment:
Will this fail only when the fail does not exist? What about missing permissions? Or are we assuming that we have those up to this point?